ui/css/perf.css

```diff
@@ -530,10 +530,10 @@ li.pagination-active.active > button {
 
 /* ReactTable stlye */
 .ReactTable .-pagination .-btn[disabled] {
-  background-color: rgba(0,0,0,0.03);
+  background-color: rgba(0, 0, 0, 0.03);
```

Comment:
We should probably move all of these styles (and in intermittent failures CSS file) to the `treeherder-global.css` file in case these libraries are used elsewhere in the future.